### PR TITLE
Base RDM Update | Break Fixes

### PR DESCRIPTION
**Pull request by Cephid**
This is a bare bones update of the RDM module with the new actions/statuses, adjustments to remove scatter and old impact as well as add Scorch to the combo module.  This is just a parity with pre 5.0.

### CODE REVIEW

File: src/data/ACTIONS/RDM.js [hunk 1]
```diff
@@ -143,10 +165,22 @@ export default {
...
+		potency: 600, //Gains 21 white, if White is lower 100% Verstone ready
 		combo: {
 			from: 7529,
-			potency: 550,
+			potency: 600,
+		},
+	},
+	SCORCH: {
+		id: 16530,
+		name: 'Verholy',
```
[the remark this answers is not in the document]
Comment (Cephid) [reply src/data/ACTIONS/RDM.js]: Bah, I thought I got all the copy/paste errors.  Thanks for the catch.

File: locale/de/messages.json [hunk 2]
```diff
@@ -688,15 +684,14 @@
   "whm.dots.rule.name": "Halte deine DoTs aufrecht",
   "whm.dots.suggestion.clip-dia.content": "",
   "whm.dots.suggestion.clip-dia.why": "{0} von {1} durch zu frühes Reaktivieren verloren.",
-  "whm.lily-cap.requirement.name": "",
-  "whm.lily-cap.suggestion.content": "",
-  "whm.lily-cap.suggestion.why": "",
   "whm.lily-blood.suggestion.content": "",
   "whm.lily-blood.suggestion.why": "",
+  "whm.lily-cap.suggestion.content": "",
+  "whm.lily-cap.suggestion.why": "",
```
[the remark this answers is not in the document]
Comment (Cephid) [reply locale/de/messages.json]: This is likely from one of the two yarn extracts I had to run while working, one after modifying all the Impact items and another when I did a second merge from upstream this morning.  I had conflicts in the translation files on the merge and had to do a full extract/build after I was done.

[the remark this answers is not in the document]
Comment (ackwell) [reply locale/de/messages.json]: Yeah, looks like some kludge coming through from extracts. Should be fine.

File: src/data/ACTIONS/RDM.js [hunk 3]
```diff
@@ -167,17 +201,27 @@ export default {
 		onGcd: true,
 		cooldown: 2.5,
 		castTime: 2,
-		potency: 240, //Increase White and Black by 3
+		potency: 250, //Increase White and Black by 3
 		breaksCombo: true,
 	},
+	// IMPACT: {
```
Comment (ackwell) [inline src/data/ACTIONS/RDM.js]: drop the old stuff when you get a chance pl0x

File: src/parser/jobs/rdm/modules/GeneralCDDowntime.js [hunk 4]
```diff
@@ -12,5 +12,6 @@ export default class GeneralCDDowntime extends CooldownDowntime {
 		ACTIONS.CORPS_A_CORPS.id,
 		ACTIONS.DISPLACEMENT.id,
 		ACTIONS.SWIFTCAST.id,
+		ACTIONS.LUCID_DREAMING.id,
```
Comment (ackwell) [inline src/parser/jobs/rdm/modules/GeneralCDDowntime.js]: maaan ain't that the truth

File: src/parser/jobs/rdm/modules/MeleeCombos.js [hunk 5]
```diff
@@ -392,18 +384,23 @@ export default class MeleeCombos extends Module {
...
-					this._endCombo()
-				} else {
-					this._currentCombo.events.push(event)
-					this._currentCombo.lastAction = event
-					if (action.combo.end) {
-						this._currentCombo.finisher = {
-							used: event.ability,
-						}
-						this._handleFinisher()
+				//console.log(util.inspect(action, {showHidden: true, depth: null}))
```
Comment (ackwell) [inline src/parser/jobs/rdm/modules/MeleeCombos.js]: try not to leave too much debug stuff around 🙏

**Pull request merged**